test/generic/UnivPoly-test.jl

```diff
@@ -6,6 +6,9 @@
          @test universal_polynomial_ring(R; internal_ordering=ord, cached = true) === universal_polynomial_ring(R; internal_ordering=ord, cached = true)
          @test universal_polynomial_ring(R; internal_ordering=ord, cached = false) !== universal_polynomial_ring(R; internal_ordering=ord, cached = false)
 
+         @test universal_polynomial_ring(R, [:x,:y]; internal_ordering=ord, cached = true)[1] === universal_polynomial_ring(R, [:x,:y]; internal_ordering=ord, cached = true)[1]
+         @test universal_polynomial_ring(R, [:x,:y]; internal_ordering=ord, cached = false)[1] !== universal_polynomial_ring(R, [:x,:y]; internal_ordering=ord, cached = false)[1]
```

Comment:
Please also add a test exercising the varnames features. So e.g.
```suggestion
         @test universal_polynomial_ring(R, :x => 1:3 ; internal_ordering=ord, cached = false)[1] !== universal_polynomial_ring(R, :x => 1:3; internal_ordering=ord, cached = false)[1]
```

Also some tests which verify the return values of these functions are right.

Reply:
E.g.
```
S2, x = universal_polynomial_ring(R, :x => 1:3; internal_ordering=ord)
```
and then test that `x` is a vector of length 3; compare its entries to `gen(S2, BLA)` calls. Verify that `S2` is of the right type and parent of the elements of `x`.

Ideally also test one of the slightly more complex constructors